x-pack/legacy/plugins/reporting/server/routes/lib/make_request_facade.test.ts

```diff
@@ -4,13 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Legacy } from 'kibana';
+import { KibanaRequest, RequestHandlerContext } from 'src/core/server';
 import { makeRequestFacade } from './make_request_facade';
 
 describe('makeRequestFacade', () => {
   test('creates a default object', () => {
-    const legacyRequest = ({
-      getBasePath: () => 'basebase',
+    const getBasePath = () => 'basebase';
+    const context = ({
+      getSavedObjectsClient: () => {},
+    } as unknown) as RequestHandlerContext;
```

Comment:
You can use `coreMock.createRequestHandlerContext` instead (`src/core/server/mocks.ts`)